learn/app-development/core/core-overview.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Core Overview"
+id: "core-overview"
+sidebar_label: "core overview"
+---
+---
+
+Core Project can be defined as a base application that contains basic functionalities that can be later used to create child applications on top of the base core application. It is similar to Project shell that is already supported in WaveMaker but offers some extended features like easy upgrade of the core application at any point of the development of the child application and so on.
```

Comment:
We should start with the usecase where core project can be used.

Does your delivery model resemble something like a core product is created which is then customed by different teams before deploying? Then creating and using a core project in WaveMaker will be a perfect solution. Lets explore how create a core project and how to extend the core project by creating child project.